### PR TITLE
Remaining Hardcoded to `master` items

### PR DESCRIPTION
**Pull request by scbedd**
Based off of [this issue](https://github.com/Azure/azure-sdk-for-python/issues/17978)

### CODE REVIEW

**Review by chidozieononiwu — APPROVED** [review]

**Comment by scbedd** [comment]
Attempting an internal run as well.

File: scripts/auto_release/main.py [hunk 1]
```diff
@@ -417,4 +417,4 @@ def main():
         my_print(e)
     else:
         with open(f'{OUT_PATH}/output.txt', 'w') as file_out:
-            file_out.writelines([f'{NEW_BRANCH}\n', "master" if TRACK == '2' else 'release/v3'])
+            file_out.writelines([f'{NEW_BRANCH}\n', "main" if TRACK == '2' else 'release/v3'])
```
Comment (scbedd) [inline scripts/auto_release/main.py:420]: @msyyc please yell at me if this is an improper update. Given the context of [this issue](https://github.com/Azure/azure-sdk-for-python/issues/18213) I think we're fine. Please correct me if I'm wrong!

**Pull request merged**